src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -29,6 +29,22 @@
         }
     ],
     "metrics": [
+        {
+            "name": "cloudwatchlogs_filter",
+            "description": "Filters a CloudWatch Logs entity.",
+            "metadata": [
+                { "type": "result" },
+                { "type": "cloudWatchResourceType" },
+                { "type": "source" },
+                { "type": "hasTextFilter", "required": false },
+                { "type": "hasTimeFilter", "required": false }
+            ]
+        },
+        {
+            "name": "cloudwatchlogs_jumpToStream",
```

Comment:
If it's not hard to plumb through a source such as "JumpTo" or "ClickedSearchResults" , does it make sense to add this as a `source` on `recordCloudwatchlogsOpen` instead of adding a dedicated metric?

Reply:
using existing definitions ,  `lambda_goToHandler` is a precedent . Could name this `cloudwatchlogs_goToStream`. 

But it's hard to argue that this is different than `cloudwatchlogs_open`, semantically. Only the source is different.

Reply:
I decided to swap to `cloudwatchlogs_open` since I don't see a strong argument for creating a new metric. If we don't need it, might as well not add it.